src/table.ts

```diff
@@ -1587,28 +1437,32 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
           : entryBatch.map(Mutation.parse),
       };
 
-      const retryOpts = {
-        currentRetryAttempt: numRequestsMade,
-        // Handling retries in this client. Specify the retry options to
-        // make sure nothing is retried in retry-request.
-        noResponseRetries: 0,
-        shouldRetryFn: (_: any) => {
-          return false;
-        },
-      };
-
       options.gaxOptions = populateAttemptHeader(
         numRequestsMade,
         options.gaxOptions
       );
+      if (
+        options.gaxOptions?.retry === undefined &&
+        options.gaxOptions?.retryRequestOptions === undefined
+      ) {
+        // For now gax will not do any retries for table.mutate unless
+        // the user specifically provides retry or retryRequestOptions in the
+        // call.
```

Comment:
Question about this comment which seems to conflict with the comment below - in this comment it says gax won't do table.mutate retries unless the user provides retry or retryRequestOptions, however, the default behavior defined in the [client config](https://github.com/googleapis/nodejs-bigtable/blob/724b7118972e6697e3fd648b5d6ce2d047ab2b6a/src/v2/bigtable_client_config.json#L42-L46) says otherwise - it will retry on the `idempotent` codes with the given backoff options there. 

In your comment below though, you mention custom logic. I do believe that if there's still custom logic in place you'll be bypassing gax, but I wanted to double check if the custom logic is why it isn't handling retries. (if not, we should talk because if gaxServerStreamingRetries is true, it should be!)

Reply:
First, just to clarify, custom logic is handling retries for mutate [here](https://github.com/googleapis/nodejs-bigtable/blob/724b7118972e6697e3fd648b5d6ce2d047ab2b6a/src/table.ts#L1614) and will still handle retries after this PR is merged responding to your second point.

Responding to your first point, below we set the `retry` property to retry options that have a `shouldRetryFn` always return false. This will make gax never retry for mutateRows calls right?

Reply:
Ah yes, I believe you're right, that should be fine for now. Thanks for the clarification